app/API/V1/Controllers/APIControllerAbstract.php

```diff
@@ -29,6 +29,11 @@ public function __construct()
 	{
 		App::register(App\API\V1\Providers\APIServiceProvider::class);
 		parent::__construct();
+
+        $this->middleware(function ($request, $next) {
```

Comment:
Should the middleware be applied to the route instead of hard-coded here? Or is this basically a fix to get the user from the auth provider?

Reply:
This should go out once I am done with the JWT investigation thing. Currently Auth:user() is null in here.